File: lab10_C#/ReservationGrpc/model/AgencyEmployee.cs
namespace Reservations.model
{
    public class AgencyEmployee : IHasID<string>
    {
        private string id;
        public string username { get; set; }
        public string password { get; set; }

        public AgencyEmployee(string id, string username, string password)
        {
            this.id = id;
            this.username = username;
            this.password = password;

        }

        public string ID
        {
            get { return id; }
            set { id = value; }
        }


        public override string ToString()
        {
            return string.Format("[Agency Employee: Id={0}, Username={1}, Password={2}]", id, username, password);
        }
    }
}

File: lab10/reservationModel/src/main/java/model/AgencyEmployee.java
package model;

public class AgencyEmployee implements HasID<String> {
    private String id;
    private String username;
    private String password;

    public AgencyEmployee(String id, String username, String password) {

        this.id = id;
        this.username = username;
        this.password = password;
    }


    @Override
    public String getID() {
        return id;
    }

    @Override
    public void setID(String newID) {
        id=newID;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}
